src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/InitMeansTestOutcomeCalculator.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.InitAssessmentResult;
+
+
+@Slf4j
+public class InitMeansTestOutcomeCalculator {
+
+    public static MeansTestOutcome getInitMeansTestOutcome(InitAssessmentResult initAssessmentResult, Boolean fullAssessmentAvailable){
+        log.debug("InitMeansTestOutcome start. Inputs: initAssessmentResult = {} fullAssessmentAvailable = {} ");
+
+        MeansTestOutcome meansTestOutcome = null;
+
+        if (initAssessmentResult == null) {
+            throw new RuntimeException("InitMeansTestOutcome: Undefined outcome for these inputs: Init Means Test " +
```

Comment:
It might be nicer if this was a custom (maybe even checked) exception so that the client can understand what is going on rather than just crashing with a 500 error